lib/xcodeproj/scheme/execution_action.rb

```diff
@@ -3,56 +3,60 @@ class XCScheme
     # This class wraps the ExecutionAction node of a .xcscheme XML file
     #
     class ExecutionAction < XMLElementWrapper
+      # @param [String] action_type
+      #        The ActionType of this ExecutionAction. One of two values:
+      #
+      #        Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction,
+      #        Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.SendEmailAction
+      #
       # @param [REXML::Element] node
       #        The 'ExecutionAction' XML node that this object will wrap.
       #        If nil, will create an empty one
       #
-      def initialize(node = nil)
-        create_xml_element_with_fallback(node, 'ExecutionAction')
+      def initialize(action_type = nil, node = nil)
```

Comment:
this is a breaking change -- the action type either needs to come after the node or be a kwarg

Reply:
I'll put `action_type` after the `node` 👍 but sorry can you clarify where this would break? This is a new class but I'm wondering now if somewhere else `node` is expected to always be the first argument?

Reply:
ah sorry in the diff it looked like a change, not net-new code